### PR TITLE
 Switch to sentry-sdk from raven (rebase existing PR, and make a release)

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.14.0] - 2020-04-24
+
```
Comment (ghost) [inline CHANGELOG.md:11]: Technically this is missing a Changed header :-p

**Review by ghost — DISMISSED** [review]
I haven't tested Sentry-sdk but I assume this is all fine - are there any ramifications for our deploys? I can't think of any, so that's grand.

File: CHANGELOG.md (hunk 1 above)
Comment (Bjwebb) [reply CHANGELOG.md:11]: Fixed, as I had to fix merge conflicts here anyway.

**Comment by Bjwebb** [comment]
I've rebased this onto the new master.

No changes to our deploys needed.

**Review by ghost — DISMISSED** [review]

**Comment by jpmckinney** [comment]
[Merge Sentry PRs](https://trello.com/c/buP3pM77/215-merge-sentry-prs)

**Comment by jpmckinney** [comment]
There's one more commit from #46 that we'll want: https://github.com/OpenDataServices/lib-cove-web/pull/46/commits/1cab469fde4a7c6a77287f1953b5410441237d0b It turns out changing `LOGGING` only affects which emails are sent to [ADMINS](https://docs.djangoproject.com/en/3.0/ref/settings/#admins) (a feature we don't use, but good to have this configured anyhow). That commit adds the necessary lines for Sentry to be quiet as well.

I deployed my fork to check that it worked. Once this is merged, we can update cove-ocds and cove-oc4ids to use the latest version of this repo.

**Comment by jpmckinney** [comment]
By the way, I made the changes to `LOGGING` in the Django projects, e.g. https://github.com/open-contracting/cove-ocds/commit/3e1f9a80dde50314244a8de87207b8f8c0c41906#diff-208e9b79d8da0cf3e3d5afa2ed26933eR123-R129

If we want to have that configured here, we can make similar changes in this PR.

**Comment by Bjwebb** [comment]
I'm happy to leave the LOGGING update where it is, for now.

**Review by ghost — APPROVED** [review]

**Pull request merged**